psd-web/spec/requests/two_factor_authentication_spec.rb

```diff
@@ -67,6 +69,11 @@
           follow_redirect!
           expect(response.body).to include("Sign out")
         end
+
+        it "marks the mobile number as verified" do
+          submit_2fa
+          expect(user.reload.mobile_number_verified).to be true
```

Comment:
Suggestion: 
Suggestion: As @StupidCodeFactory would say... Rspec provides nice helpers for this 
```suggestion
          expect(user.reload).to be_mobile_number_verified
```

Reply:
I personally like the rspec helpers, but only where they make linguistic sense. "be mobile number verified" feels a bit awkward and ungrammatical to me?